registry/auth/silly/access_test.go

```diff
@@ -5,7 +5,9 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/docker/distribution/registry/api/v2"
```

Comment:
you ok with me just forcing the http status code to be 401 instead of using an errorCode?